Sources/MongoSwift/MongoCollection.swift

```diff
@@ -133,4 +133,97 @@ public class MongoCollection<T: Codable> {
 
         return try body(collection)
     }
+
+    /**
+     * Starts a `ChangeStream` on a collection. By default, the type `CollectionType` is associated with the
+     * `fullDocument` field in `ChangeStreamDocument`s emitted by the returned `ChangeStream`.
+     * - Parameters:
+     *   - pipeline: An array of aggregation pipeline stages to apply to the events returned by the change stream.
+     *   - options: An optional `ChangeStreamOptions` to use when constructing the change stream.
+     *   - session: An optional `ClientSession` to use with this change stream.
+     * - Returns: A `ChangeStream` on a specific collection.
+     * - Throws:
+     *   - `ServerError.commandError` if an error occurs on the server while creating the change stream.
+     *   - `ServerError.commandError` if the pipeline passed is invalid.
+     *   - `UserError.invalidArgumentError` if the options passed formed an invalid combination.
+     *   - `UserError.invalidArgumentError` if the `_id` field is projected out of the change stream documents by the
+     *     pipeline.
+     * - SeeAlso:
```

Comment:
maybe include your system collections see also link in all methods since it applies to all of them